src/components/DocumentRenderer/ValidityBanner/ValidityBannerContent.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { FunctionComponent } from "react";
+import { CheckStatus } from "./types";
+import { View } from "react-native";
+
+interface ValidityBannerContent {
+  checkStatus?: CheckStatus;
+  isExpanded?: boolean;
+}
+
+export const ValidityBannerContent: FunctionComponent<ValidityBannerContent> = ({
+  checkStatus = "checking",
+  isExpanded = false,
+  children
+}) => {
+  let backgroundColor;
+  switch (checkStatus) {
+    case "valid":
+      backgroundColor = "#EFFAF4";
+      break;
+    case "invalid":
+      backgroundColor = "#FCF3F3";
+      break;
+    case "checking":
+    case "unknown":
```

Comment:
Was there an `unknown` state?

Reply:
I'll remove this for now, since we don't use it.